doc/sphinx/history.rst

```diff
@@ -1396,7 +1396,7 @@ Tactics
 Extraction (See details in plugins/extraction/CHANGES and README):
 
 - An experimental Scheme extraction is provided.
-- Concerning Ocaml, extracted code is now ensured to always type-check,
+- Concerning Ocaml, extracted code is now ensured to always type check,
```

Comment:
Since you are already modifying this line:

```suggestion
- Concerning OCaml, extracted code is now ensured to always type check,
```

Reply:
We are literally rewriting history!